codefeedr-plugins/codefeedr-travis/src/main/scala/org/codefeedr/plugins/travis/TravisProtocol.scala

```diff
@@ -0,0 +1,109 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package org.codefeedr.plugins.travis
+
+import java.time.LocalDateTime
+import org.codefeedr.pipeline.PipelineItem
+import org.codefeedr.plugins.github.GitHubProtocol.PushEvent
+
+object TravisProtocol {
+
+  case class PushEventFromActiveTravisRepo(pushEventItem: PushEvent) extends PipelineItem
```

Comment:
Why do you have this wrapper class instead of just using a PushEvent?

Reply:
This is to make sure TravisPushEventBuildInfoTransformStage only receives push events from active repositories

Reply:
But you can also filter just PushEvents right?